### PR TITLE
add/react-native-flatboard

### PR DESCRIPTION
**Pull request by ArunGovil**
# 📝 Why & how
Added react-native-flatboard, a Flatlist based onboarding screen for React Native.

# ✅ Checklist
- [X] Added library to **`react-native-libraries.json`**

### CODE REVIEW

**Comment by Simek** [comment]
Hello @ArunGovil, thank you for the contribution! 👍 Beside skipping `npmPkg` field, entry look fine, failure is due to schema validation.

I will need to add the `webp` to the allowed images format after testing that those images works as expected.

File: react-native-libraries.json [hunk 1]
```diff
@@ -9053,5 +9053,13 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/ArunGovil/react-native-flatboard",
+    "npmPkg": "react-native-flatboard",
```
Comment (Simek) [inline react-native-libraries.json:9059]:
```suggestion
```

This field can be skipped, since package name matches the GitHub repository.

Comment (ArunGovil) [reply react-native-libraries.json]: Thank you @Simek for clarification. npmPkg field has been removed.

File: react-native-libraries.json [hunk 2]
```diff
@@ -9054,6 +9054,13 @@
     "web": true,
     "expo": true
   },
+  {
+    "githubUrl": "https://github.com/ArunGovil/react-native-flatboard",
+    "images": ["https://raw.githubusercontent.com/ArunGovil/react-native-flatboard/main/src/assets/demo.webp"],
```
Comment (Simek) [inline react-native-libraries.json:9059]:
```suggestion
    "images": ["https://raw.githubusercontent.com/ArunGovil/react-native-flatboard/main/src/assets/cover.webp"],
```

It looks like `demo` file no longer exist.

**Comment by ArunGovil** [comment]
@Simek Updated, looks good now.

File: react-native-libraries.json (hunk 2 above)
Comment (ArunGovil) [reply react-native-libraries.json]: Updated.

**Review by Simek — APPROVED** [review]
Thank you for the quick updates!

**Pull request merged**